src/gui/mzroll/eicwidget.cpp

```diff
@@ -55,6 +55,7 @@ EicWidget::EicWidget(QWidget *p) {
     _mouseEndPos = _mouseStartPos = QPointF(0,0); //TODO: Sahil, added while merging eicwidget
 
 	connect(scene(), SIGNAL(selectionChanged()), SLOT(selectionChangedAction()));
+    connect(this, &EicWidget::eicUpdated, this, &EicWidget::setGallaryToEics);
```

Comment:
@rish9511 Let's just correct the spelling of 'Gallery' throughtout the code base. Even the tooltip spells it 'Gallary'.